ultralytics/engine/model.py

```diff
@@ -94,7 +94,13 @@ def __init__(self, model: Union[str, Path] = 'yolov8n.pt', task=None) -> None:
             self._load(model, task)
 
     def __call__(self, source=None, stream=False, **kwargs):
-        """Calls the 'predict' function with given arguments to perform object detection."""
+        """Calls the predict() method with given arguments to perform object detection."""
+        return self.predict(source, stream, **kwargs)
+
+    def embed(self, source=None, stream=False, **kwargs):
+        """Calls the predict() method and returns concatenated embeddings."""
```

Comment:
Probably also good to add independent docstrings. This asumes the user already knows about and remembers the arguments and kwargs of  predict function.